contracts/Splitter.sol

```diff
@@ -7,34 +7,29 @@ contract Splitter {
     
     address payable[] internal recipients = [0x20601F1Ddb44E28b1511EdBD316A368D80116408, //"Bob";
                                             0x24F1500890505ceD8534502ab403F20ce99feea0]; //"Carol";
+    uint InitialValue = 100 ether;
 
     event Deposit(uint date,
                   address indexed _from,
                   address indexed _to,
```

Comment:
It looks weird that there is only one `to` here because two people are receiving money, no?